<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>perfectPersonalData</class>
 <widget class="QWidget" name="perfectPersonalData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>323</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="savePushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保 存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closePushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>关 闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>136</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>个人简介</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>22</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>昵称</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="synopsisTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="NicknameLineEdit">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>22</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>22</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sexComboBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>22</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string>男</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>男</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>女</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>未知</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>98</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>联系方式</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>98</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>98</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>邮箱</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EmailLneEdit">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>98</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="birthDateTimeEdit">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>60</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>生日</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
